### ISSUE TITLE
Not snap to the grid when Snapping is disabled

### ISSUE THREAD

**Issue by Lonsfor** [SeamusFinlayson/owlbear-measure#6]
Similarly to the regular ruler, if the toggle for snapping is off then the segmentable ruler should not snap to the grid 

![image](https://github.com/user-attachments/assets/dd595cbd-de3c-4fa2-b119-b9915a3baae2)

I also noticed that when the grid is set to hexagon then it doesnt snap but that seems unintended

**Comment by SeamusFinlayson** [comment]
I don't think that setting is actually available through the API so I don't have a way to check it. I left snapping off in the hex euclidean mode because I thought people might want that behavior so yes that is intentional. I don't really use euclidean so I don't really have strong opinions on this.
